test/metabase/lib/binning_test.cljc

```diff
@@ -31,7 +31,7 @@
     (is (= {:bin-width 12.5, :min-value 15, :max-value 27.5}
            (lib.binning/resolve-bin-width query col-quantity 15)))))
 
-(deftest ^:parallel binning-and-bucketing-only-show-up-for-returned-and-breakoutable-columns
+(deftest ^:parallel binning-and-bucketing-only-show-up-for-returned-and-breakout-columns
```

Comment:
Why the rename?

Reply:
because technically `breakoutable-columns` do not contain `:unit` and `:binning` now. But `breakout-column` does.